### DISCUSSION TITLE
two card of same state officer on prod.

### DISCUSSION THREAD

**Discussion by himanshugoel435** [Sunbird-cQube/community#847] [General]
Hello,

I am encountering an issue where I am getting one state officer locally, but two state officers in the production environment. Please help me resolve this issue.
![image](https://github.com/Sunbird-cQube/community/assets/141122499/c4acafb7-1849-4c4e-ad09-3593831ace4b)

**Comment by RNarmatha** [comment]
@pandutibil - Could you please look into this isue

**Comment by pandutibil** [comment]
Okay @RNarmatha, Checking on that

**Comment by pandutibil** [comment]
@himanshugoel435 Did you made any changes in the file rbacConfig.ts which is there in this location: "dashboard-ms/src/app/shared/components/rbac-dialog/rbacConfig.ts"

**Reply by himanshugoel435** [reply]
no @pandutibil

**Comment by pandutibil** [comment]
Okay, can we connect once?

**Reply by himanshugoel435** [reply]
yes

**Comment by pandutibil** [comment]
Please join here: [https://meet.google.com/omb-sraz-vek](https://meet.google.com/omb-sraz-vek)

**Comment by pandutibil** [comment]
Configured invalid config type and state code in the environment file of dashboard-ms. So, updated them with the valid values and the issue has been fixed

**Comment by RNarmatha** [comment]
Thanks Pandu for the update
